front/tests/pathfinding-local.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { test } from '@playwright/test';
+import { PlaywrightHomePage } from './home-page-model';
+import PlaywrightRollingstockModalPage from './rollingstock-modal-model';
+import PlaywrightMap from './map-model';
+import VARIABLES from './assets/operationStudies/test_variables';
+import PlaywrightScenarioPage from './scenario-page-model';
+
+if (!process.env.CI) {
+  test.describe('Testing pathfinding in a local environment', () => {
+    let playwrightHomePage: PlaywrightHomePage;
+    let playwrightScenarioPage: PlaywrightScenarioPage;
+
+    test.beforeEach(async ({ page }) => {
+      playwrightHomePage = new PlaywrightHomePage(page);
+      playwrightScenarioPage = new PlaywrightScenarioPage(page);
+
+      await playwrightHomePage.goToHomePage();
+
+      // Real click on project, study, scenario
+      await playwrightHomePage.goToOperationalStudiesPage();
+      await playwrightHomePage.page
+        .getByTestId('_@Test integration project')
+        .locator('div')
+        .getByRole('button')
+        .click();
+
+      await playwrightHomePage.page
+        .getByTestId('_@Test integration study')
+        .getByRole('button')
+        .click();
+
+      await playwrightHomePage.page
+        .getByTestId('_@Test integration scenario')
+        .getByRole('button')
+        .click();
+      await playwrightScenarioPage.checkInfraLoaded();
+      await playwrightHomePage.page.getByTestId('scenarios-add-train-schedule-button').click();
+    });
+
+    test('Testing pathfinding electrification error', async () => {
```

Comment:
It's always good to describe why the test is failing. Ex: `Test pathfinding: no electrified rolling stock for this path throws error`.

Reply:
Done thank you